python/strings_udf/strings_udf/__init__.py

```diff
@@ -26,11 +65,11 @@ def _get_appropriate_file(sms, cc):
         return None
 
 
-def maybe_patch_numba_linker(driver_version):
+def maybe_patch_numba_linker(driver_version, ptx_version):
```

Comment:
Only a minor nit, but it's not really the PTX version you're passing in here, it's the toolkit version for the PTX - maybe `ptx_toolkit_version` is slightly closer to what the parameter is. Or, `min_driver_version` is what it represents.